### PR TITLE
Convert GimmePeers to AngleSharp

### PR DESCRIPTION
**Pull request by cadatoiva**
Untested
#7292

### CODE REVIEW

File: src/Jackett.Common/Indexers/GimmePeers.cs [hunk 1]
```diff
@@ -141,17 +143,17 @@ private async Task ProcessPage(List<ReleaseInfo> releases, string searchUrl)
             var results = response.Content;
             try
             {
-                CQ dom = results;
+                var parser = new HtmlParser(); var dom = parser.ParseDocument(results);
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/GimmePeers.cs]: new line

File: src/Jackett.Common/Indexers/GimmePeers.cs [hunk 2]
```diff
@@ -161,26 +163,26 @@ private async Task ProcessPage(List<ReleaseInfo> releases, string searchUrl)
                     }
 
                     // Check if the release has been assigned a category
-                    if (row.Cq().Find("td:eq(0) a").Length > 0)
+                    if (row.QuerySelector("td:nth-of-type(1) a") != null)
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/GimmePeers.cs]: assign `row.QuerySelector("td:nth-of-type(1) a")`to a var to reuse in the next line

Comment (cadatoiva) [reply src/Jackett.Common/Indexers/GimmePeers.cs]: Not sure if ReSharper or built in to VS, but CTRL+R,V (refactor into variable) detected both and made this as simple as naming the variable in one place.

Comment (ngosang) [reply src/Jackett.Common/Indexers/GimmePeers.cs]: I'm using Rider, but thank you anyway!

Comment (cadatoiva) [reply src/Jackett.Common/Indexers/GimmePeers.cs]: If it's a feature of ReSharper, then it should be integrated in Rider, since that's the same company.

**Pull request merged**